### PR TITLE
df-daemon: change the import path

### PR DESCRIPTION
**Pull request by zhouhaibing089**
basically, as a go project, it should be discoverable by go tools,
thus I makde a change to update its import path to be:

  github.com/alibaba/Dragonfly/src/daemon/src/df-daemon

with this change, you can run:

  go get github.com/alibaba/Dragonfly/src/daemon/src/df-daemon

and get a compiled binary in your $GOPATH/bin.

### CODE REVIEW

**Comment by zhouhaibing089** [comment]
cc @allencloud PTAL.

**Comment by allencloud** [comment]
Yes, we need to use gofmt, golint or some other tools to ensure the code quality. 
While I am not the one who is responsible for dragonfly. @zhouhaibing089 
But you can still find me in dragonfly's community.

**Comment by zhouhaibing089** [comment]
@allencloud Thanks, yeah, there are many things to do in order to make it great(though it looks great already according to the numbers).

**Comment by zhouhaibing089** [comment]
@garfield009 could you please take a look?

File: src/daemon/src/df-daemon/initializer/initializer.go [hunk 1]
```diff
@@ -69,7 +71,7 @@ func cleanLocalRepo() {
 						log.Warnf("walk file:%s error:%v", path, err)
 					} else {
 						if info.Mode().IsRegular() {
-							if time.Now().Unix() - reflect.ValueOf(info.Sys()).Elem().FieldByName("Atim").Field(0).Int() >= 3600 {
+							if time.Now().Unix()-reflect.ValueOf(info.Sys()).Elem().FieldByName("Atim").Field(0).Int() >= 3600 {
```
Comment (allencloud) [inline src/daemon/src/df-daemon/initializer/initializer.go]: I think this block of code are not so good. I recommend that we should use more gotools to guarantee the code format, lint, vet and some more other things.

Comment (zhouhaibing089) [reply src/daemon/src/df-daemon/initializer/initializer.go]: There are more bits needs to be revisited, which I would like to do several PRs as a follow up, and make those changes gradually.

The tooling part which I believe we can track it as an issue.

Comment (garfield009) [reply src/daemon/src/df-daemon/initializer/initializer.go]: ok,you can do it,I will review your pull request as soon as possible.

**Pull request merged**